Helper/OmiseHelper.php

```diff
@@ -20,7 +20,7 @@
 use Omise\Payment\Model\Config\Rabbitlinepay;
 use Omise\Payment\Model\Config\Ocbcpao;
 use Omise\Payment\Model\Config\Grabpay;
-use Omise\Payment\Model\Config\Cc as Config;
+use Omise\Payment\Model\Config\Config;
```

Comment:
There's one usage that uses `Config` as `Cc`, so we may need to change it to `Cc::CODE`.

https://github.com/omise/omise-magento/blob/c3e5ab2218c60c704335aaecc79edf594b084e2e/Helper/OmiseHelper.php#L84